doc/api/typescript.md

```diff
@@ -68,6 +68,19 @@ By intentionally not supporting syntaxes that require JavaScript code
 generation, and by replacing inline types with whitespace, Node.js can run
 TypeScript code without the need for source maps.
 
+The suggested `tsconfig.json` settings for type stripping are:
+
+```json
+{
+  "compilerOptions": {
+     "module": "nodenext",
+     "allowImportingTsExtensions": true,
+     "rewriteRelativeImportExtensions": true,
```

Comment:
I'm [not seeing any documentation](https://www.typescriptlang.org/tsconfig/) for `rewriteRelativeImportExtensions`.

Which version of TS was this introduced in?

And should we mention the minimum version of TS in the docs here?

Reply:
It's not out yet; due in 5.7: https://devblogs.microsoft.com/typescript/announcing-typescript-5-7-beta/#path-rewriting-for-relative-paths

Reply:
I see. I made a suggestion to mention 5.7 or newer above 👍